presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -244,7 +244,8 @@ public int getMaxTotalRunningTaskCount()
         return maxTotalRunningTaskCount;
     }
 
-    @Config("experimental.max-total-running-task-count")
+    @Config("max-total-running-task-count-to-kill-query")
+    @LegacyConfig("experimental.max-total-running-task-count")
```

Comment:
Isn't the idea of "experimental" configs is that they can change without notice and therefore don't require @LegacyConfig-style transition?